Sources/Sync/SyncCoordinator/DTO/Campaign.swift

```diff
@@ -0,0 +1,397 @@
+//
+//  Campaign.swift
```

Comment:
I think pulling this out into a separate file makes sense. Can we do the same with geofences and beacons (currently inlined in the `GeofenceSyncParticipant` file)? What you're really defining here is the structure of the "node" so perhaps a more descriptive name for the folder would be `Nodes` instead of `DTO`?

Reply:
done